components/dom_crawler.rst

```diff
@@ -282,6 +282,60 @@ and :phpclass:`DOMNode` objects:
 
         $html = $crawler->html();
 
+Expression Evaluation
+~~~~~~~~~~~~~~~~~~~~~
+
+The ``evaluate()`` method evaluates the given XPath expression.
+The return value depends on if the expression operates on simple values
+(like HTML attributes), or returns a subset of the current document.
```

Comment:
"[...] if the expression operates on simple values (like HTML attributes) or a subset of the current document." ?